src/Security/Authentication/JwtBearer/src/JwtBearerConfigureOptions.cs

```diff
@@ -72,7 +72,7 @@ public void Configure(string? name, JwtBearerOptions options)
             ValidAudiences = audiences,
             ValidAudience = audience,
             ValidateIssuerSigningKey = true,
-            IssuerSigningKeys = GetIssuerSigningKeys(configSection, issuers),
+            IssuerSigningKeys = GetIssuerSigningKeys(configSection, [issuer, ..issuers]),
```

Comment:
The fact that we didn't pass the standalone issuer here was always a bug, correct?

Reply:
It was a regression caused in .NET 9 by #52821. I plan to backport this once approved.